.github/issue_template.md

```diff
@@ -0,0 +1,21 @@
+# Issue type
+- Camayoc bug
+- Enhancement
+- Other
+
+# Description of issue
+
+# Completion checklist
+To close this issue, the following items must be satisfied:
+- [ ] One
+- [ ] Two
+
+# Related development issues
+
+### User stories
+
+### Enhancements
+
+### Bugs
+
+### Documentation
```

Comment:
Suggested boilerplate text for the ###Documentation section (if my assumptions about the intended usage are correct):
Provide a summary of the documentation changes/additions required for each affected part of the documentation. For example, installation, config/administration, user, troubleshooting. If practical/possible, include more specific information, e.g., the locations of the required changes.

Reply:
I like the idea for the documentation, but I am wondering if the changes about quipucords documentation should go into quipucords issues. I think it does not make sense to have quipucords documentation changes listed on camayoc's repository.

IMO quipucords user stories could have a section called _Acceptance Criteria_ which would include a check list or some items to make that user story accepted. For example:

- [ ] Update section X of Y Document stating that feature A will now have a new option called `new option`. This option can accept the following values 1, 2, 3.
- [ ] The CLI should expose the option `new option` for the command `qpc command` and only accept the valid values (listed on the above item)
- [ ] The UI should present a new form field which will be a a choice field with the values listed on the first item.

So each item in the acceptance criteria could link to an issue where the related work to that is/will be tracked.

With that information, the QE team can pull the latest changes and if that user story is closed we can check if docs are updated, CLI and UI interfaces are updated and say we are ready to ship that user story.

Reply:
I guess I meant the documentation section to be links to related docs about the features we were testing.

I did update the quipucords issue template to have acceptance criteria:
https://github.com/quipucords/quipucords/blob/master/.github/issue_template.md#acceptance-criteria